Solutions/CohesitySecurity/Tools/setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/zsh
+
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+. ./error_handler.sh
+. ./json_parser.sh
+
+. ./prerequisite.sh
+error_handler "Failed to complete prerequisites."
+
+. ./deploys.sh
+error_handler "Failed to deploy resources."
+
+. ./config.sh
+error_handler "Failed to configure resources."
+
+. ./test.sh
```

Comment:
A 'setup.sh' should not include test.sh. Please rename the file appropriately or remove this invocation here and create a new file called run.sh which calls both setup.sh and test.sh

Reply:
just renamed to sentinel_provisioning.sh